Dockerfile.sle12-sp3

```diff
@@ -18,10 +18,12 @@ RUN zypper ar -f -p 95 http://download.opensuse.org/repositories/YaST:/SLE-12:/S
 # see https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#run
 # https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#/build-cache
 # why we need "zypper clean -a" at the end
+## FIXME: remove the ruby2.4 workaround (replace by pure "ruby")
 RUN zypper --gpg-auto-import-keys --non-interactive in --no-recommends \
-  ruby && zypper clean -a
+  ruby2.4 && zypper clean -a
 
-RUN RUBY_VERSION=`rpm --eval '%{rb_default_ruby_abi}'` && \
+## FIXME: remove the 2.4 workaround, use RUBY_VERSION=`rpm --eval '%{rb_default_ruby_abi}'`
+RUN RUBY_VERSION=ruby:2.4.0 && \
```

Comment:
For SP3 we still use the TW image because the 42.3 image is not yet available at the docker hub. I guess it will be available **after** the official 42.3 release.

See https://hub.docker.com/_/opensuse/